static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -0,0 +1,249 @@
+    Math.abs(+moment(event.dateReceived) - +moment(event.dateCreated)) > latencyThreshold;
+
+  const hasPreviousEvent = defined(event.previousEventID);
+  const hasNextEvent = defined(event.nextEventID);
+
+  const quickTrace = useContext(QuickTraceContext);
+
+  return (
+    <CarouselAndButtonsWrapper>
+      <StyledButtonBar merged>
+        <Button
+          size={BUTTON_SIZE}
+          icon={<IconPrevious size={BUTTON_ICON_SIZE} />}
+          aria-label="Oldest"
+          to={{
+            pathname: `${baseEventsPath}oldest/`,
+            query: {...location.query, referrer: 'oldest-event'},
+          }}
+          disabled={!hasPreviousEvent}
+        />
+        <Button
+          size={BUTTON_SIZE}
+          icon={<IconChevron direction="left" size={BUTTON_ICON_SIZE} />}
+          aria-label="Older"
+          to={{
+            pathname: `${baseEventsPath}${event.previousEventID}/`,
+            query: {...location.query, referrer: 'previous-event'},
+          }}
+          disabled={!hasPreviousEvent}
+        />
+        <EventLabelContainer>
+          <div>
+            <strong>Event ID:</strong> {getShortEventId(event.id)}{' '}
+            {(event.dateCreated ?? event.dateReceived) && (
+              <EventTimeLabel>
+                (
+                <TimeSince
+                  date={event.dateCreated ?? event.dateReceived}
+                  tooltipBody={<EventCreatedTooltip event={event} />}
+                  unitStyle="short"
+                />
+                {isOverLatencyThreshold && (
+                  <Tooltip title="High latency">
+                    <StyledIconWarning size="xs" color="warningText" />
+                  </Tooltip>
+                )}
+                )
+              </EventTimeLabel>
+            )}
+          </div>
+        </EventLabelContainer>
+        <Button
+          size={BUTTON_SIZE}
+          icon={<IconChevron direction="right" size={BUTTON_ICON_SIZE} />}
+          aria-label="Newer"
+          to={{
+            pathname: `${baseEventsPath}${event.nextEventID}/`,
+            query: {...location.query, referrer: 'next-event'},
+          }}
+          disabled={!hasNextEvent}
+        />
+        <Button
+          size={BUTTON_SIZE}
+          icon={<IconNext size={BUTTON_ICON_SIZE} />}
+          aria-label="Newest"
+          to={{
+            pathname: `${baseEventsPath}latest/`,
+            query: {...location.query, referrer: 'latest-event'},
+          }}
+          disabled={!hasNextEvent}
+        />
+      </StyledButtonBar>
+      <DropdownMenu
+        position="bottom-end"
+        triggerProps={{
+          'aria-label': t('Event Actions Menu'),
+          icon: <IconEllipsis size={BUTTON_ICON_SIZE} />,
+          showChevron: false,
+          size: BUTTON_SIZE,
+        }}
+        items={[
+          {
+            key: 'copy-event-url',
+            label: t('Copy Event Link'),
+            onAction: () =>
+              copyToClipboard(
```

Comment:
@malwilley let's get some analytics on the actions like copying

Reply:
Added analytics to all the actions in the overflow menu